### PR TITLE
fix(Prominent): Fix prominent theme colors for Movistar

### PR DESCRIPTION
**Pull request by jmpg93**
Hi there!

I'm fixing prominent variant colors for Movistar using the [latests definition of design](https://github.com/Telefonica/mistica-design/blob/5cfbe888c44a25145c136398a56a2726d254ba1e/tokens/movistar.json#L846).
It broke while aligning the colors of mistica with android and web a few weeks ago.

### CODE REVIEW

File: Mistica/Source/Colors/MovistarColorPalette.swift [hunk 1]
```diff
@@ -17,10 +17,10 @@ struct MovistarColors: Colors {
     let backgroundOverlay = MovistarColorPalette.grey6.withAlphaComponent(0.6) | MovistarColorPalette.darkModeGrey.withAlphaComponent(0.8)
     let backgroundSkeleton = MovistarColorPalette.grey1 | MovistarColorPalette.grey6
     let backgroundSkeletonInverse = MovistarColorPalette.movistarBlue55 | MovistarColorPalette.grey6
-    let navigationBarBackground = MovistarColorPalette.movistarBlue | MovistarColorPalette.darkModeBlack
+    var navigationBarBackground: UIColor { MovistarColorPalette.movistarBlueWithProminentVariant | MovistarColorPalette.darkModeBlack }
```
Comment (jmpg93) [inline Mistica/Source/Colors/MovistarColorPalette.swift:20]: We need to use computed properties. Otherwise, the prominent color will be wrong.

Comment (salavert) [reply Mistica/Source/Colors/MovistarColorPalette.swift:20]: should we state some kind of premises so other will have this problem into account in the future?

Comment (jmpg93) [reply Mistica/Source/Colors/MovistarColorPalette.swift:20]: What do you mean?

**Review by alexanegon — APPROVED** [review]

File: Mistica/Source/Colors/MovistarColorPalette.swift (hunk 1 above)
Comment (jmpg93) [reply Mistica/Source/Colors/MovistarColorPalette.swift:20]: Talked offline. The best option for this to not happen again is to create tests for prominent. I've create this ticket to https://jira.tid.es/browse/IOS-7587 tackle it down at some point.

**Review by salavert — APPROVED** [review]

**Pull request merged**